bluetooth_mesh/messages/silvair/emergency_lighting_test.py

```diff
@@ -0,0 +1,262 @@
+)
+
+EmergencyLightingTestServerDurationTestGet = Struct()
+
+EmergencyLightingTestServerDurationTestStart = Struct()
+
+EmergencyLightingTestServerDurationTestStop = Struct()
+
+EmergencyLightingTestServerDurationTestStatusMinimal = Struct(
+    "tai_timestamp" / TimestampAdapter(Timestamp),
+    "execution_status" / EnumAdapter(Int8ul, TestExecutionStatus),
+    "execution_result" / ExecutionResult,
+    "duration_result" / Int16ul,
+)
+
+EmergencyLightingTestServerDurationTestStatusOptional = Struct(
+    Embedded(EmergencyLightingTestServerDurationTestStatusMinimal),
+    "relative_timestamp" / Int32ul,
+)
+
+EmergencyLightingTestServerDurationTestStatus = NamedSelect(
+    optional=EmergencyLightingTestServerDurationTestStatusOptional,
+    minimal=EmergencyLightingTestServerDurationTestStatusMinimal,
+)
+
+EmergencyLightingTestServerPropertyId = FieldAdapter(
+    Select(
+        EnumAdapter(Int16ul, PropertyID),
+        Int16ul
+    ),
+    Int16ul
+)
+
+EmergencyLightingTestServerPropertyGet = Struct(
+    "property_id" / EmergencyLightingTestServerPropertyId,
+)
+
+PropertyValue = Switch(
+    this.property_id,
+    PropertyDict,
+)
+
+EmergencyLightingTestServerPropertySet = Struct(
+    "property_id" / EmergencyLightingTestServerPropertyId,
+    "value" / PropertyValue,
+)
+
+EmergencyLightingTestServerPropertyStatus = Struct()
+
+
+EmergencyLightingTestServerParams = SwitchStruct(
+    "subopcode" / EnumAdapter(Int8ul, EmergencyLightingTestServerSubOpcode),
+    "payload" / Switch(
+        this.subopcode,
+        {
+            EmergencyLightingTestServerSubOpcode.ELT_FUNCTIONAL_TEST_GET: EmergencyLightingTestServerFunctionalTestGet,
+            EmergencyLightingTestServerSubOpcode.ELT_FUNCTIONAL_TEST_START: EmergencyLightingTestServerFunctionalTestStart,
+            EmergencyLightingTestServerSubOpcode.ELT_FUNCTIONAL_TEST_STOP: EmergencyLightingTestServerFunctionalTestStop,
+            EmergencyLightingTestServerSubOpcode.ELT_FUNCTIONAL_TEST_STATUS: EmergencyLightingTestServerFunctionalTestStatus,
+            EmergencyLightingTestServerSubOpcode.ELT_DURATION_TEST_GET: EmergencyLightingTestServerDurationTestGet,
+            EmergencyLightingTestServerSubOpcode.ELT_DURATION_TEST_START: EmergencyLightingTestServerDurationTestStart,
+            EmergencyLightingTestServerSubOpcode.ELT_DURATION_TEST_STOP: EmergencyLightingTestServerDurationTestStop,
+            EmergencyLightingTestServerSubOpcode.ELT_DURATION_TEST_STATUS: EmergencyLightingTestServerDurationTestStatus,
+            EmergencyLightingTestServerSubOpcode.ELT_PROPERTY_GET: EmergencyLightingTestServerPropertyGet,
+            EmergencyLightingTestServerSubOpcode.ELT_PROPERTY_SET: EmergencyLightingTestServerPropertySet,
+            EmergencyLightingTestServerSubOpcode.ELT_PROPERTY_SET_UNACKNOWLEDGED: EmergencyLightingTestServerPropertySet,
+            EmergencyLightingTestServerSubOpcode.ELT_PROPERTY_STATUS: EmergencyLightingTestServerPropertySet,
```

Comment:
`EmergencyLightingTestServerPropertyStatus`

Reply:
Format is the same

Reply:
So why we have this?

`EmergencyLightingTestServerPropertyStatus = Struct()`

Reply:
:229

Reply:
Because I forgot to remove it 🫣